CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 - Add custom [`ocamllsp/getDocumentation`](/ocaml-lsp-server/docs/ocamllsp/getDocumentation-spec.md) request (#1336)
 
+- Add custom [`ocamllsp/merlinJump`](/ocaml-lsp-server/docs/ocamllsp/merlinJump-spec.md) request (#1374)
+
```

Comment:
This is not the correct log section, it should be part of the # Unreleased section.